.vscode/launch.json

```diff
@@ -12,13 +12,13 @@
             "webRoot": "${workspaceFolder}"
         },
         {
-            "name": "Attach to Windows Docker container",
+            "name": "Attach to Docker container",
             "type": "node",
             "request": "attach",
             "port": 9229,
             "protocol": "inspector",
-            "localRoot": "${workspaceFolder}",
-            "remoteRoot": "C:\\app"
+            "localRoot": "${workspaceFolder}/apps/backend",
+            "remoteRoot": "/app"
```

Comment:
This gets rid of our old windows container debug profile and replaces it with the newer one that our debug instructions on confluence advise putting here. Once this merges in, that section will need to be updated https://user-office-project.atlassian.net/wiki/spaces/UOP/pages/19136712/STFC+development+workflow#Debugging-the-backend